apps/console/src/features/users/models/user.ts

```diff
@@ -343,6 +344,29 @@ export interface PayloadInterface {
       schemas: string[];
 }
 
+/**
+ * Type of the bulk user import operation status.
+ */
+export type BulkUserImportOperationStatus = "Success" | "Failed" | "Warning";
```

Comment:
Isn't this same as `BulkUserImportStatus` defined in apps/console/src/features/users/constants/user-management-constants.ts? shall we keep one of the definitions and remove the other?

Reply:
Removed this in https://github.com/wso2/identity-apps/pull/6597.